ios/RCTPSPDFKit/RCTPSPDFKitViewManager.m

```diff
@@ -107,6 +121,42 @@ @implementation RCTPSPDFKitViewManager
 
 RCT_EXPORT_VIEW_PROPERTY(onStateChanged, RCTBubblingEventBlock)
 
+RCT_CUSTOM_VIEW_PROPERTY(availableFontNames, NSArray, RCTPSPDFKitView) {
+  if (json && [RCTConvert NSArray:json]) {
+    view.availableFontNames = [RCTConvert NSArray:json];
+    staticAvailableFontNames = view.availableFontNames;
+
+    // We use a subclassed font picker view controller to customize it.
+    [view.pdfController updateConfigurationWithoutReloadingWithBuilder:^(PSPDFConfigurationBuilder * _Nonnull builder) {
+      [builder overrideClass:PSPDFFontPickerViewController.class withClass:CustomFontPickerViewController.class];
+    }];
```

Comment:
Could we always use the custom override, so we don't need to update the configuration all the time one of these properties is set, and just fallback to the default behavior, when none of these variables are set?

Reply:
Good point! 👍 Addressed in https://github.com/PSPDFKit/react-native/pull/363/commits/cc92d1ad000c4af1b4951752b84e4f5e0734c4bd